Microsoft.Toolkit.Uwp.UI/Helpers/VisualHelpers.cs

```diff
@@ -0,0 +1,56 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using Windows.Foundation.Metadata;
+using Windows.UI;
+
+namespace Microsoft.Toolkit.Uwp.UI.Helpers
+{
+    /// <summary>
+    /// Helpers for Handling UWP Visuals.
+    /// </summary>
+    public static class VisualHelpers
```

Comment:
Let's not add new apis with this PR - please move this to the sample app. This seems specific to the sample app anyway

Reply:
Moved.